base/deprecated.jl

```diff
@@ -1815,6 +1815,13 @@ end
     @deprecate get_creds!(cache::CachedCredentials, credid, default) get!(cache, credid, default)
 end
 
+# PR #23755
+# we need to use depwarn() directly since the function is not exported
```

Comment:
You can do
```
@deprecate runtests test false
```

Reply:
Hmm okay, that could probably be fixed so that when `export` is false, the full name is printed.